parsers/MD.py

```diff
@@ -210,107 +164,62 @@ def fetch_price(
             "This parser is not yet able to parse past dates for price"
         )
 
-    dt = datetime.now(tz=TZ)
+    dt = datetime.now(timezone.utc)
     return template_price_response(zone_key, dt, 145.0)
 
 
+@refetch_frequency(timedelta(days=2))
 def fetch_consumption(
     zone_key: str = "MD",
     session: Session | None = None,
     target_datetime: datetime | None = None,
     logger: Logger = getLogger(__name__),
 ) -> list[dict] | dict:
     """Requests the consumption (in MW) of a given country."""
-    if target_datetime:
-        archive_data = get_archive_data(session, target_datetime)
-
-        datapoints = []
-        for entry in archive_data:
-            datapoint = template_consumption_response(
-                zone_key, entry.datetime, entry.consumption
-            )
-            datapoints.append(datapoint)
-        return datapoints
-    else:
-        field_values = get_data(session)
-
-        consumption = field_values[other_fields[0]["index"]]
-
-        dt = datetime.now(tz=TZ)
-
-        datapoint = template_consumption_response(zone_key, dt, consumption)
+    archive_data = get_archive_data(target_datetime, session=session, backlog_days=1)
 
-        return datapoint
+    datapoints = []
+    for entry in archive_data:
+        datapoint = template_consumption_response(
+            zone_key, entry.datetime, entry.consumption
+        )
+        datapoints.append(datapoint)
+    return datapoints
 
 
+@refetch_frequency(timedelta(days=2))
```

Comment:
question: would this refetch once every two days or twice a day?

Reply:
every 20 minutes ;)

This is just so that when we manually refetch large sections of data we create one request for every 2 days worth of data instead of 1 per hour or whatever the default is.